proto/profile/network.proto

```diff
@@ -0,0 +1,298 @@
+// Copyright(c) 2025 Zededa, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+import "google/protobuf/timestamp.proto";
+import "evecommon/evecommon.proto";
+import "config/netcmn.proto";
+import "config/netconfig.proto";
+
+package org.lfedge.eve.profile;
+
+option go_package  = "github.com/lf-edge/eve-api/go/profile";
+option java_package = "org.lfedge.eve.profile";
+
+// NetworkInfo is sent by EVE in a POST request to the `api/v1/network` endpoint.
+// It publishes multiple configurations for each network port, including:
+//   - The latest configuration received from the controller.
+//   - The fallback configuration used when controller connectivity is lost
+//     or the latest configuration is invalid.
+//   - Any locally modified configuration (e.g., from LPS) that overrides
+//     controller or fallback settings.
+//
+// For each port, the `NetworkPortConfig.config_applied` flag indicates which
+// of these configurations is currently active on the device.
+message NetworkInfo {
+  // The most recent network configuration received from the controller.
+  NetworkConfig latest_config = 1;
+
+  // Current status of controller connectivity and the state of fallback configuration.
+  NetworkConfigTestingStatus config_testing = 2;
+
+  // Contains only the ports with locally modified configurations and/or associated
+  // error messages. Errors may result from LPS-provided configuration being invalid
+  // or from changes not being permitted for the given port.
+  // This represents the subset of the configuration that differs from
+  // the controller-provided configuration due to changes requested by LPS.
+  // Ports not listed here are using either the latest controller config
+  // or the fallback config, as indicated by `config_testing`.
+  NetworkConfig local_changes = 3;
+}
+
+// Reports the current status of controller connectivity testing and fallback configuration usage.
+message NetworkConfigTestingStatus {
+  // True if the device currently has connectivity to the controller.
+  bool controller_reachable = 1;
+
+  // Describes the last connectivity failure; empty if connectivity is fine.
+  string connectivity_error = 2;
+
+  // Timestamp of the most recent controller connectivity test.
+  google.protobuf.Timestamp last_test_time = 3;
+
+  // True if EVE is currently using a fallback configuration due to controller connectivity loss.
+  bool is_using_fallback_config = 4;
+
+  // The fallback network configuration currently in use.
+  // Empty if not using fallback configuration.
+  NetworkConfig fallback_config = 5;
```

Comment:
It feels a bit odd to included these two fields in the testing status message. Why not put them in the NetworkInfo message?